doc/classes/OS.xml

```diff
@@ -824,6 +825,9 @@
 			If [code]true[/code], the window is minimized.
 		</member>
 		<member name="window_per_pixel_transparency_enabled" type="bool" setter="set_window_per_pixel_transparency_enabled" getter="get_window_per_pixel_transparency_enabled">
+			If [code]true[/code], the window background is transparent and window frame is removed.
+			Use [code]get_tree().get_root().set_transparent_background(true)[/code] to disable main viewport background rendering.
+			Note: This property has no effect if "Project &gt; Project Settings &gt; Display &gt; Window &gt; Per-pixel transparency &gt; Allowed" setting is disabled.
```

Comment:
I think you could use `[member ProjectSettings.display/window/per_pixel_transparency/allowed]` to reference the property directly with a hyperlink to its documentation.

Reply:
It works, but I don't know which one is better? Project setting is most likely set manually and link is not very readable. A normal text with attached hyperlink would be better, but AFAIK it's not supported.

![Screenshot 2019-06-04 at 23 56 34](https://user-images.githubusercontent.com/7645683/58913309-96cb7780-8724-11e9-99c4-c671ea8ea3d3.png)

Also, all existing references to the project settings use this plain text format, eg.:
https://github.com/godotengine/godot/blob/master/doc/classes/RigidBody2D.xml#L174